README.md

```diff
@@ -1,2 +1,180 @@
+  <ul>
+    <li>JSX</li>
+    <li>CRA</li>
+    <li>Jest</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Render</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Stock Market Most Gainers API</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Use GitFlow.**
+- **Single page React app.**
+- **Consume app data from API.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Project Presntation -->
+
+## :movie_camera: Project Presentation <a name="project-presentation"></a>
+
+Walking through portfolio outline.
+
+- [Project Presentation Link](https://www.loom.com/share/78f59f913310492aacaa81867070289d?sid=8b836f4b-8878-494a-8404-a18097c00637)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Most-Gainers Live Demo](https://most-gainers.onrender.com/)  //
+*Press `Ctrl + click` to open the link in a new tab.*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install Node.js.
+- Visual Studio Code or equivalent. 
+- Internet Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd my-folder
+- git clone [https://github.com/Abdelaziz-Mahdi/Most-Gainers.git](https://github.com/Abdelaziz-Mahdi/Most-Gainers.git)
+
+### Install
+
+Install this project with:
+
+- cd my-project
+- git config --global --add safe.directory my-directory/my-folder/my-project
+ 
+### Usage
+
+To run the project, execute the following command:
+
+- code .
+- npm install
+- npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+npm test
+
+### Deployment
+
+You can deploy this project using the following:
+
+- Render.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Abdelaziz Ali
+
+- GitHub: [@Abdelaziz-Mahdi](https://github.com/Abdelaziz-Mahdi)
+- Twitter: [@_AbdelazizMahdi](https://twitter.com/_AbdelazizMahdi)
+- LinkedIn: [in/abdelaziz-mahdi](https://www.linkedin.com/in/abdelaziz-mahdi)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Template used in the project provided by (Microverse)!
+- Original design idea by Nelson Sakwa on Behance.
```

Comment:
- [ ] Kindly add the link to the author of the design on behance and the project